common/src/main/java/com/revenuecat/purchases/common/caching/DeviceCache.kt

```diff
@@ -63,6 +56,8 @@ open class DeviceCache(
         "$apiKeyPrefix.purchaserInfoLastUpdated"
     }
 
+    private val offeringsResponseCacheKey: String by lazy { "$apiKeyPrefix.offeringsResponse" }
```

Comment:
This currently lives in the default shared preferences... We probably want to think about having our own shared preferences file though, to avoid devs touching it accidentally.

Reply:
Yeah iOS defaults to a custom `UserDefaults` now.